src/main/java/com/iri/training/repository/UserCommentRepositoryImpl.java

```diff
@@ -35,26 +35,28 @@ public UserCommentRepositoryImpl() throws IOException {}
 	@Override
 	public List<UserComment> getCommentsByUserId(final Long userId) throws SQLException {
 
-		logger.debug("ENTERED getUserCommentById: " + userId);
+		logger.debug("ENTERED getUserCommentById for userId: " + userId);
 
 		String sql=property.getString("SELECT_COMMENT");
 		jdbcTemplate=new JdbcTemplate(dataSource);
 		final List<UserComment> userComment=jdbcTemplate.query(sql,new Object[]{userId},new UserCommentMapper());
-		logger.debug("EXITING getCommentsByUserId: " + userComment);
+
+		logger.debug("RETRIEVED " + userComment.size() + " comments");
+		logger.debug("EXITING getCommentsByUserId for userId: " + userId);
 		return userComment;
 
 	}
 
 	@Override
 	public UserComment createUserComment(final UserComment userComment) throws SQLException {
 
-		logger.debug("ENTERED createUserComment: " + userComment);
+		logger.debug("ENTERED createUserComment for comment: " + userComment.toString());
+
```

Comment:
No need to call toString, it is called automatically from log4j, and it will throw a null pointer exception if no comment is returned from DB